serveradmin/servershell/views.py

```diff
@@ -379,12 +379,15 @@ def new_object(request):
 @login_required
 def clone_object(request):
     try:
+        cloned_attributes = list(Attribute.specials)
+        # intern_ip must be uniq - this would always conflict on save
```

Comment:
I think this comment is misleading. intern_ip must not be unique, we have a lot of loadbalancers which share a intern_ip with another loadbalancer. I still does make sense to empty this attribute during cloning.

Reply:
Good catch! I found that out later and already adjusted the commit message but forgot about the comment in the code.